http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -544,6 +544,19 @@ object CommonTestData {
     Some(0)
   )
 
+  val default2CreateAzureRuntimeReq = CreateAzureRuntimeRequest(
```

Comment:
Nit: if you'd like, you can `.copy` on the original  `defaultCreateAzureRuntimeReq` and just change the disk name.

I.E. (just replace `diskReq` with actual param name)
defaultCreateAzureRuntimeReq.copy(diskReq = defaultCreateAzureRuntimeReq.diskReq.copy(diskName = AzureDiskName("diskName2")))
`

Reply:
Ohh, this is a lot nicer indeed, I will do that, thanks for the tip :)